CMakeLists.txt

```diff
@@ -149,6 +151,8 @@ if(TRITON_ENABLE_CC_HTTP OR TRITON_ENABLE_CC_GRPC OR TRITON_ENABLE_PERF_ANALYZER
       -DTRITON_ENABLE_CC_GRPC:BOOL=${TRITON_ENABLE_CC_GRPC}
       -DTRITON_ENABLE_PERF_ANALYZER:BOOL=${TRITON_ENABLE_PERF_ANALYZER}
       -DTRITON_ENABLE_PERF_ANALYZER_C_API:BOOL=${TRITON_ENABLE_PERF_ANALYZER_C_API}
+      -DTRITON_ENABLE_TF_SERVING:BOOL=${TRITON_ENABLE_TF_SERVING}
+      -DTRITON_ENABLE_TORCHSERVE:BOOL=${TRITON_ENABLE_TORCHSERVE}
```

Comment:
I think you might need to add these steps to L192 too?

Reply:
Thanks for the catch! Updated.